agent/agent-tooling/build.gradle.kts

```diff
@@ -42,7 +45,7 @@ dependencies {
     exclude("org.ow2.asm", "asm")
   }
 
-  //  compileOnly("io.opentelemetry:opentelemetry-sdk-extension-tracing-incubator")
+  compileOnly("io.opentelemetry:opentelemetry-sdk-extension-tracing-incubator:1.17.0-alpha")
```

Comment:
we should use the version that corresponds with the SDK that we're using (1.36.0)

Reply:
that's the latest from maven central though

![image](https://github.com/microsoft/ApplicationInsights-Java/assets/56097766/6251d658-3ec3-4af8-b3ba-0bf7f54418aa)

Reply:
it must have been renamed then to something else since then, check out https://github.com/open-telemetry/opentelemetry-java?tab=readme-ov-file#releases or the release notes to see what it's been renamed to